### PR TITLE
CT 107: Improve profile question clarity and workflow for international institutions #19703

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
As a Comparison Tool user, I need the ability to specify attendance at an international campus so that I can get an accurate benefits estimate when viewing an institution profile age.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19703
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19760
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19779

## Testing done
- local testing

## Acceptance criteria
- [ ] If the campus has extensions that are a non-us physical location, they are displayed in the extension drop down list following the question: "Choose the location where you'll take your classes".
     - Institution Name (City, Country)
- [ ] The  question text "Please enter the zip code where you'll take your classes" is replaced with:
     - "If you're taking classes in the U.S., enter the location's zip code" when the profile being viewed is an international location
     - There is no change to the question text when the profile being viewed is a US or US territory location.
- [ ] A checkbox is displayed under the input field for the zip code entry that allows the user to specify that they are selecting an international location that is not listed.
     - The checkbox label is: "I'll be taking classes outside of the U.S. and U.S. territories"
- [ ] When the checkbox is checked: 
     - the ZIP code question (AC # 2) and input box is hidden.
     - the benefit estimator is updated to display the international housing VA national average rate when the answer to "Did you use your Post-9/11 GI Bill benefits for tuition, housing, or books for a term that started before January 1, 2018?" is yes.
     - the benefit estimator is updated to display the international housing DOD national average rate when the answer to "Did you use your Post-9/11 GI Bill benefits for tuition, housing, or books for a term that started before January 1, 2018?" is no.
- [ ] When the checkbox is unchecked, the ZIP code question (AC # 2) and input box is displayed.
     - the benefit estimator is updated to display the estimated benefits based on the rate of the institution profile being viewed 
- [ ] In the "School Locations" section, international locations display "City, Country" in the location column.
- [ ] In the "School Locations"  section, international locations display the international housing rate in the "Estimated Housing" column.
     - the international housing VA national average rate when the answer to "Did you use your Post-9/11 GI Bill benefits for tuition, housing, or books for a term that started before January 1, 2018?" is yes.
     - the  international housing DOD national average rate when the answer to "Did you use your Post-9/11 GI Bill benefits for tuition, housing, or books for a term that started before January 1, 2018?" is no.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by delli-gatti-michael** [comment]
Local testing complete for #19760. Waiting for other stories included in this PR to be tested before approving.

**Review by sonntag-adam — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/gi/components/profile/CalculatorForm.jsx [hunk 1]
```diff
@@ -571,15 +589,45 @@ class CalculatorForm extends React.Component {
       const errorMessageCheck =
         errorMessage !== '' ? errorMessage : inputs.beneficiaryZIPError;
 
-      amountInput = (
+      // Prod Flag for 19703
+      if (environment.isProduction() || !inputs.classesOutsideUS) {
+        // Prod Flag for 19703
```
Comment (U-DON) [inline src/applications/gi/components/profile/CalculatorForm.jsx]: Accidental copy of the commment above?

**Review by U-DON — APPROVED** [review]

File: src/applications/gi/components/profile/CalculatorForm.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/gi/components/profile/CalculatorForm.jsx]: comment is for each `environment.isProduction()` usage, could have been reduced to one

**Pull request merged**